src/components/settings/ContractVersion/index.tsx

```diff
@@ -42,6 +44,24 @@ export const ContractVersion = ({ isGranted }: { isGranted: boolean }) => {
           Unsupported contract
         </Typography>
       )}
+      <div style={{ marginTop: 15 }}>
+        {needsUpdate ? (
+          <Tooltip
+            title="Update now to take advantage of new features and the highest security standards available.
+You will need to confirm this update just like any other transaction."
+            placement="right-start"
+          >
+            <Typography variant="body2" style={{ display: 'flex', alignItems: 'center' }}>
+              <SvgIcon component={InfoIcon} inheritViewBox fontSize="small" color="border" style={{ marginRight: 4 }} />
```

Comment:
By using Emotion's `sx` prop, we get access to the [spacing provided by our theme](https://github.com/safe-global/web-core/blob/dev/src/styles/theme.ts#L64). This is also the spacing we follow for other tooltip icons:

```suggestion
              <SvgIcon
                component={InfoIcon}
                inheritViewBox
                fontSize="small"
                color="border"
                sx={{
                  verticalAlign: 'middle',
                  mr: 0.5,
                }}
              />
```